Cargo.toml

```diff
@@ -963,6 +966,11 @@ name = "determine_if_only_one_instance_is_running"
 path = "src/determine_if_only_one_instance_is_running.rs"
 
 [[bin]]
+# http://rosettacode.org/wiki/Digital_root
+name = "digital_root"
+path = "src/digital_root.rs"
```

Comment:
@kwaegel Yeah I feel you there. :( We had similar problems on Mac.

This helped:

```
export OPENSSL_INCLUDE_DIR=`brew --prefix openssl`/include
export OPENSSL_LIB_DIR=`brew --prefix openssl`/lib
```

But it obviously won't work on WInders.